src/clicast.ts

```diff
@@ -0,0 +1,264 @@
+import * as vscode from "vscode";
+
+import {
+  updateDataForEnvironment,
+} from "./helper"
+import {
+  openMessagePane,
+  vcastMessage,
+  vectorMessage,
+} from "./messagePane";
+import {
+  getEnviroPathFromID,
+  getTestNode,
+  testNodeType,
+} from "./testData";
+import { updateTestPane } from "./testPane";
+import {
+  commandStatusType,
+  executeCommand,
+  exeFilename,
+} from "./utilities";
+import { getClicastArgsFromTestNode } from "./vcastTestInterface";
+import { getEnviroNameFromScript } from "../src-common/commonUtilities";
+
+
+const fs = require("fs");
+const os = require("os");
+const path = require("path");
+const spawn = require("child_process").spawn;
+
+
+const clicastName = "clicast";
+export let clicastCommandToUse: string | undefined = undefined;
+
+const vcastqtName = "vcastqt";
```

Comment:
Can we call this package something else, given it isn't _just_ `clicast`?

Reply:
yea, I considered that, but the only vcastqt thing is the name, will re-consider

Reply:
Will change to vcastUtilities.py

Reply:
Done